api_app/api/routes/migrations.py

```diff
@@ -43,6 +47,10 @@ async def migrate_database(resources_repo=Depends(get_repository(ResourceReposit
         migration_status = "Executed" if workspace_migration.moveAuthInformationToProperties() else "Skipped"
         migrations.append(Migration(issueNumber="PR 1726", status=migration_status))
 
+        logging.info("#1406 - Extra field to support UI")
+        resource_migration.add_deployment_status_field(operations_repo)
+        migrations.append(Migration(issueNumber="1406", status="Executed"))
```

Comment:
Maybe put something more useful than always "Executed" (e.g. number of rows updated)